manager/dispatcher/dispatcher_test.go

```diff
@@ -460,14 +461,20 @@ func TestAssignmentsInitialNodeTasks(t *testing.T) {
 	}
 
 	// updating all the tasks will attempt to remove all the secrets for the tasks that are in state > running
-	err = gd.Store.Update(func(tx store.Tx) error {
-		for _, task := range tasks {
-			assert.NoError(t, store.UpdateTask(tx, task))
-		}
-		return nil
-
-	})
-	assert.NoError(t, err)
+	assert.NoError(t, testutils.PollFuncWithTimeout(nil, func() error {
```

Comment:
Even if there was, it would not be able to change the task status in betweenthe `GetTask` and the `UpdateTask`. `Update` transactions are atomic because they take an update lock on the store.